judge/signals.py

```diff
@@ -41,10 +41,8 @@ def problem_update(sender, instance, **kwargs):
                        for lang, _ in settings.LANGUAGES])
     cache.delete_many(['generated-meta-problem:%s:%d' % (lang, instance.id) for lang, _ in settings.LANGUAGES])
 
-    if hasattr(settings, 'DMOJ_PDF_PROBLEM_CACHE'):
```

Comment:
Why are we getting rid of this if?

Reply:
`DMOJ_PDF_PROBLEM_CACHE` is in `settings.py`, so it would always evaluate to true. (https://github.com/DMOJ/site/blob/master/dmoj/settings.py#L73)